rollup.config.js

```diff
@@ -164,21 +164,31 @@ const testBuild = {
     }),
     wasm(wasmOptions.browser)
   ]
-};
+});
 
-export default Object.assign([
+/**
+ * Rollup CLI supports custom options; their name must start with `config`,
+ * e.g. see `--configDebug` example at
+ * https://rollupjs.org/command-line-interface/#configuration-files
+ *
+ * The custom options we support are:
+ * @param commandLineArgs.configBuildOnly {'dist'|'node'|'lightweight'|'test'} to specify a build target;
+ *    defaults to 'dist', which does not build tests.
```

Comment:
Strictly speaking, this option is more flexible than that, e.g. you can also pass `node/openpgp.cjs` to only build that file, and so on.